infrastructure/conf/nginx-conf-local/default

```diff
@@ -31,6 +32,10 @@ server {
     location = /en/communities/digital-talent { return 301 https://www.canada.ca/en/government/system/digital-government/gcdigital-community.html; }
     location = /fr/communities/digital-talent { return 301 https://www.canada.ca/fr/gouvernement/systeme/gouvernement-numerique/collectivite-gcnumerique.html; }
 
+    # permanent redirect within the same domain with relative path
+    absolute_redirect off;
```

Comment:
It looks like [this directive can be used in the context of a location block](http://nginx.org/en/docs/http/ngx_http_core_module.html#absolute_redirect).  It would make me more comfortable if you issued it in these two locations rather than at the top level where it could affect other rules.